.github/workflows/browserstack-workflow.yml

```diff
@@ -0,0 +1,60 @@
+name: BrowserStack
+
+# BrowserStack tests are run against the deployed GitHub pages URL
+# The closest event we can use to trigger when these are updated is page_build.
+# on: [page_build]
+on: [push]
+
+jobs:
+  browserstack:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - config: 'Windows_10_latest'
+            browser: 'chrome'
+            browserstack_username: 'PRO_BROWSERSTACK_USERNAME'
+            browserstack_access_key: 'PRO_BROWSERSTACK_ACCESS_KEY'
+          - config: 'Windows_10_latest'
+            browser: 'edge'
+            browserstack_username: 'PRO_BROWSERSTACK_USERNAME'
+            browserstack_access_key: 'PRO_BROWSERSTACK_ACCESS_KEY'
+          - config: 'Windows_7_latest'
+            browser: 'firefox'
+            browserstack_username: 'PRO_BROWSERSTACK_USERNAME'
+            browserstack_access_key: 'PRO_BROWSERSTACK_ACCESS_KEY'
+          - config: 'OSX_Mojave_latest'
+            browser: 'safari'
+            browserstack_username: 'PRO_BROWSERSTACK_USERNAME'
+            browserstack_access_key: 'PRO_BROWSERSTACK_ACCESS_KEY'
+          - config: 'iPhone11_13'
+            browser: 'ios'
+            browserstack_username: 'MOBILE_BROWSERSTACK_USERNAME'
+            browserstack_access_key: 'MOBILE_BROWSERSTACK_ACCESS_KEY'
+          - config: 'iPhone8_12'
+            browser: 'ios'
+            browserstack_username: 'MOBILE_BROWSERSTACK_USERNAME'
+            browserstack_access_key: 'MOBILE_BROWSERSTACK_ACCESS_KEY'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+      - run: bundle exec rake design_system:all
+        env:
+          BROWSERSTACK: true
```

Comment:
can't these values be read from the env file we provide?

Reply:
Not really, this is the documented way of doing this for an action; to pass things in via this env property. If we go with this long term than that env file would go anyway.